src/qt/optionsmodel.cpp

```diff
@@ -88,9 +88,10 @@ void OptionsModel::Init(bool resetSettings)
     // by command-line and show this in the UI.
 
     // Main
-    if (!settings.contains("nDatabaseCache"))
-        settings.setValue("nDatabaseCache", (qint64)nDefaultDbCache);
-    if (!m_node.softSetArg("-dbcache", settings.value("nDatabaseCache").toString().toStdString()))
+   std::string nDbCacheSettings = settings.contains("nDatabaseCache") ?
+     settings.value("nDatabaseCache").toString().toStdString() :
+     std::to_string(nDefaultDbCache);
```

Comment:
whitespace issue here, you can use `git diff -U0 HEAD~3.. | ./contrib/devtools/clang-format-diff.py -p1 -i -v` to fix: https://github.com/bitcoin/bitcoin/blob/master/contrib/devtools/README.md#clang-format-diffpy

Reply:
good one, thanks!